### ISSUE TITLE
machine stats not displaying

### ISSUE THREAD

**Issue by juliangruber** [geut/permanent-seeder#137]
on `GET /api/stats/host`, this is returned:

```
top.time is not a function
```

from the logs:

```
    TypeError: top.time is not a function
        at Service.handler (/home/ubuntu/.npm-global/lib/node_modules/@geut/permanent-seeder/node_modules/@geut/permanent-seeder-sdk/src/services/metrics.service.js:85:23)
        at async Service.handler (/home/ubuntu/.npm-global/lib/node_modules/@geut/permanent-seeder/node_modules/@geut/permanent-seeder-sdk/src/services/metrics.service.js:112:23)
        at async Service.handler (/home/ubuntu/.npm-global/lib/node_modules/@geut/permanent-seeder/node_modules/@geut/permanent-seeder-sdk/src/services/metrics.service.js:29:9)
        at async Promise.all (index 0)
        at async Promise.all (index 0)
```

**Comment by juliangruber** [comment]
The HTTP status code was 501 (Not implemented) although I think it should be 500 (Internal Server Error)

**Comment by dpaez** [comment]
Hi @juliangruber thanks for reporting this issue. It should be fixed now. 

Closed with #138
